README.md

```diff
@@ -122,7 +122,11 @@ err := reader.Read(func(msg Message) error{
 defer reader.Close()
 ```
 
-Writer with brokers hosts and topic
+#####Dead letter queue
+
+If you need to save messages that couldn't be processed, you have to use constructor NewReaderWithDLQ which takes name of DLQ topic as additional parameter.
```

Comment:
a would add a link to the constructor within `NewReaderWithDLQ` word. This would be convenient for others to quick lookup to the func. Especially that we don't add the info about that if `dlqTopic` is empty it will be set as `topic.dlc` instead.